config/jest/config.json

```diff
@@ -12,6 +12,7 @@
     "<rootDir>/src/demos"
   ],
   "moduleNameMapper": {
+    "@uifabric/utilities/(.*)": "<rootDir>/node_modules/@uifabric/utilities/",
```

Comment:
Don't understand why this is working, cause you're mapping a module to the same module...